app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt

```diff
@@ -56,7 +58,8 @@ class KaliumConfigsModule {
             guestRoomLink = BuildConfig.ENABLE_GUEST_ROOM_LINK,
             wipeOnCookieInvalid = BuildConfig.WIPE_ON_COOKIE_INVALID,
             wipeOnDeviceRemoval = BuildConfig.WIPE_ON_DEVICE_REMOVAL,
-            wipeOnRootedDevice = BuildConfig.WIPE_ON_ROOTED_DEVICE
+            wipeOnRootedDevice = BuildConfig.WIPE_ON_ROOTED_DEVICE,
+            isWebSocketEnabledByDefault = runBlocking { !context.isGoogleServicesAvailable() }
```

Comment:
We can replace it with a build time flag, and set it to true for specific builds

Reply:
I also thought that, but after discussing with Marco/Leen and pairing the old app behavior, this is the way they decided to go. funny fact the old app also relied in a build config, but was changed overtime.